TestResultSummaryService/package.json

```diff
@@ -23,7 +23,7 @@
     "body-parser": "^1.19.0",
     "compression": "^1.7.4",
     "cors": "^2.8.5",
-    "csv": "^2.0.0",
+    "csv": "^5.1.3",
```

Comment:
Is this required? Please don't change major version unless needed to avoid accidentally breaking things.